src/app/components/saved_playlists/saved_playlists.ui

```diff
@@ -9,14 +9,26 @@
         <property name="vscrollbar-policy">always</property>
         <property name="min-content-width">250</property>
         <child>
-          <object class="GtkFlowBox" id="flowbox">
-            <property name="margin-start">8</property>
-            <property name="margin-end">8</property>
-            <property name="margin-top">8</property>
-            <property name="margin-bottom">8</property>
-            <property name="min-children-per-line">1</property>
-            <property name="selection-mode">none</property>
-            <property name="activate-on-single-click">0</property>
+          <object class="GtkOverlay" id="overlay">
+            <child>
+              <object class="GtkFlowBox" id="flowbox">
+                <property name="margin-start">8</property>
+                <property name="margin-end">8</property>
+                <property name="margin-top">8</property>
+                <property name="margin-bottom">8</property>
+                <property name="min-children-per-line">1</property>
+                <property name="selection-mode">none</property>
+                <property name="activate-on-single-click">0</property>
+              </object>
+            </child>
+            <child type="overlay">
+              <object class="AdwStatusPage" id="status_page">
+                <property name="title" translatable="yes" comments="A title that is shown when the user has not saved any playlists.">You have no saved playlists</property>
+                <property name="description" translatable="yes" comments="A description of what happens when the user has saved playlists.">Your playlists will be shown here.</property>
+                <property name="icon-name">emblem-music-symbolic</property>
+                <property name="visible">false</property>
```

Comment:
it might need to be true initially (first login for instance)

Reply:
Indeed, it makes more sense